.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.wordWrap": "on"
```

Comment:
Should this file be added to .gitignore?

Reply:
Yes. This is VS code setting. Will add to gitignore